### ISSUE TITLE
deploying migration profile

### ISSUE THREAD

**Issue by PeterP55P** [OracleSiebel/ConfiguringSiebel#55]
This is more a question about whether you tried to create and deploy migration profile on your setup. So far I was unable to successfully deploy the migration profile and I cannot find the reason what's wrong. When I was checking HTTP trace I can see that there is something blocking the POST request. I did several combinations of ports and hosts in the profile configuration but none of them were success. I'll be glad for any hint as in logs I haven't found any useful info. If you didn't try this yet and not planing to do it then you can close this.

**Comment by OracleSiebel** [comment]
Haven't yet tried. But would like to make sure it works and also have an automated method via REST. Let's keep this open. How are you orchestrating the containers?

**Comment by PeterP55P** [comment]
I've expanded your script to have more servers and I also have another env. running on Windows (single box with all servers running on one VM). What I'm trying to achieve is to migrate from one env to another. The adjusted scripts are able automatically build env with 2 APP servers and 3 CGWs with 1 SAI. That all is combination of REST and Bash (I'm following what you created and just expanding it more). Migration profile and deploy I'm doing manually unless I'll  be able to make it work. Then I'll prepare REST for it.

**Comment by PeterP55P** [comment]
seems I just found what was the issue... it turns out that I have to strictly follow lower case and capital letters in hostname. my hostname is composed of both but I used only lower case. now I changed it and deployment is green now. I suspect this is related to certificate but I'm not sure as from my experiences certificates shouldn't be checking the case of the letters. the other thing which comes to my mind is that it might be due to linux which cares about the lower/upper case.  all in all migration profile is deployed now. I will check that more.

**Comment by OracleSiebel** [comment]
Thanks for the update. I think I did work around that with this docker run parameter:

https://github.com/OracleSiebel/ConfiguringSiebel/blob/master/Containerization/Docker/ol7/manage/scripts/automate-architecture/generic/container/startCGW#L40

This should make sure that the lower case version of the hostname is always available within the network, and I did this for all containers. However, my use case was limited to having everything in a single network, so I can see how you might have run into this. It seems that at some point during the setup, the hostname is retrieved and converted to lower case, resulting in the break you've experienced.
